libs/eo/transfers/src/lib/eo-create-transfer-agreement-modal.component.ts

```diff
@@ -129,22 +129,27 @@ export class EoTransfersCreateModalComponent {
 
     this.creatingTransferAgreementProposal = true;
     this.proposalId = null;
-    const proposal = {
-      receiverTin: receiverTin,
+    const senderOrganization: Actor | undefined = this.actors().find(
+      (actor) => actor.tin === transferAgreementFormValues.senderTin
+    );
+    const proposal: TransferAgreementProposalRequest = {
+      senderOrganizationId: senderOrganization?.org_id as string,
       startDate,
-      endDate,
-      transferAgreementStatus: 'Proposal' as EoListedTransfer['transferAgreementStatus'],
+      endDate: endDate ?? 0,
```

Comment:
is 0 == empty/null?

Reply:
the double question marks is a way of nullguarding in typescript, where you can give a default value, for when the endDate is null or undefined :)